include/fmt/format.h

```diff
@@ -389,7 +389,7 @@ template <typename To, typename From, FMT_ENABLE_IF(sizeof(To) > sizeof(From))>
 inline auto bit_cast(const From& from) -> To {
   constexpr auto size = static_cast<int>(sizeof(From) / sizeof(unsigned));
   struct data_t {
-    unsigned value[size];
+    unsigned value[static_cast<size_t>(size)];
```

Comment:
Let's use to_unsigned instead of static_cast

Reply:
Sure :)
In case I did something badly, feel free to fix it directly and close this PR. ;)

Reply:
@vitaut ``to_unsigned`` is constexpr only since C++14.
I think simple ``static_cast`` is better here.

Reply:
Right. I switched back to cast and merged.